tests/html/await-fetch.html

```diff
@@ -0,0 +1,10 @@
+<script>
```

Comment:
Where is this test is played?

Reply:
I used the file for manual tests. Removed now.